frontend/src/metabase/home/homepage/components/HomeCaption/HomeCaption.styled.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import { breakpointMinExtraLarge } from "metabase/styled-components/theme";
+
+export interface CaptionProps {
+  primary?: boolean;
+}
+
+export const CaptionRoot = styled.div<CaptionProps>`
+  display: flex;
+  align-items: center;
+  color: ${props =>
+    props.primary ? color("text-dark") : color("text-medium")};
```

Comment:
@mazameli I'm not sure if it  was intended to have different color for different sections, e.g. for x-rays and recents/popular items, please check this

Reply:
Yes, thanks for the attention to detail! It's intentional because the heading for x-rays is more important for the user to read to understand what they're looking at, whereas the other headings are just "supporting" text.